src/inverseRobot.js

```diff
@@ -7,7 +7,26 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+  const arrKeys = Object.keys(robot);
+  const arrValues = Object.values(robot);
+  const uniqueValues = [];
+
+  for (const char of arrValues) {
+    if (!uniqueValues.includes(char)) {
+      uniqueValues.push(char);
+    }
+  }
```

Comment:
you don't need logic with arrays, you can create loop for the object `robot` and do everything only with newRobot and robot objects